tests/helpers.py

```diff
@@ -155,8 +180,7 @@ def create_ceph_block_pool(pool_name=None):
 
 
 def create_storage_class(
```

Comment:
missing some parameters from #337 pr plz fix this might fail tests/manage/test_verify_rbd_based_rwo_pvc.py script

Reply:
Yes, will rebase